libsolidity/codegen/YulUtilFunctions.cpp

```diff
@@ -1282,32 +1275,35 @@ string YulUtilFunctions::decreaseByteArraySizeFunction(ArrayType const& _type)
 string YulUtilFunctions::increaseByteArraySizeFunction(ArrayType const& _type)
 {
 	string functionName = "byte_array_increase_size_" + _type.identifier();
-	return m_functionCollector.createFunction(functionName, [&]() {
+	return m_functionCollector.createFunction(functionName, [&](vector<string>& _args, vector<string>&) {
+		_args = {"array", "data", "oldLen", "newLen"};
```

Comment:
Hmm, why do we have `_args` as an input which gets overwritten here?

Reply:
I guess that's the new `createFunction` interface?

Reply:
Yes, that was discussed here https://github.com/ethereum/solidity/pull/11066
At one time I wanted to return struct, but we ended up with output arguments

Reply:
Thanks, wasn't aware!